src/ray/common/ray_config_def.h

```diff
@@ -690,7 +690,7 @@ RAY_CONFIG(std::string, predefined_unit_instance_resources, "GPU")
 /// The scheduler will treat these custom resource types as unit_instance.
 /// Default custom_unit_instance_resources is "neuron_cores".
```

Comment:
```suggestion
/// Default custom_unit_instance_resources is "neuron_cores,TPU".
```

Is this right? Feel free to rewrite the whole comment if you understand it, it's kind of confusing to me at the moment.

Reply:
Great point, I've rewritten the comment to match my understanding